x-pack/plugins/security_solution/public/detection_engine/rule_management/logic/coverage_overview/models/mitre_tactic.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { CoverageOverviewRuleData } from './rule_data';
+import type { CoverageOverviewMitreTechnique } from './mitre_technique';
+
+export interface CoverageOverviewMitreTactic {
```

Comment:
Does it make sense to also have a type for representing the whole coverage data for the page?
When calling the API endpoint, we could normalize its response to this type: either at the API client's level or inside the business logic.

Reply:
Added in 94a7f1cc24c7f948d04bcaf4f4a4bcf5baf0de7b.

Reply:
Perfect, thank you!